src/mod_last.erl

```diff
@@ -55,8 +55,7 @@
 -export([config_metrics/1]).
 
 -ignore_xref([
-    behaviour_info/1, on_presence_update/5, process_local_iq/4,
-    process_sm_iq/4, remove_user/3, session_cleanup/5, remove_domain/3
+    behaviour_info/1, process_local_iq/4, process_sm_iq/4
```

Comment:
This line can be deleted as: 
* `behaviour_info/1` don't need to be ignored,
* `process_local_iq/4` and `process_sm_iq/4` don't exist.